meta-refkit-core/recipes-ostree/refkit-ostree/refkit-ostree_0.0.bb

```diff
@@ -0,0 +1,42 @@
+SUMMARY = "IoT RefKit ostree helper, scripts, services, et al."
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE-BSD;md5=f9f435c1bd3a753365e799edf375fc42"
+
+SRC_URI = " \
+  file://${PN} \
+"
+
+DEPENDS = "ostree"
+
+inherit autotools pkgconfig systemd distro_features_check
+
+REQUIRED_DISTRO_FEATURES = "ostree systemd"
+
+S = "${WORKDIR}/${PN}"
+
+PACKAGES += "${PN}-initramfs"
+
+FILES_${PN}-initramfs = " \
+    ${bindir}/refkit-ostree \
+"
+
+FILES_${PN} = " \
+    ${bindir}/refkit-ostree-update \
```

Comment:
I split out the useful bits to a separate commit, adding a new (initial version of an) OSTree howto and updating the existing system-update doc with slight adjustments and a pointer to the howto. Couldn't really squash the doc bits to the original C-ifying/repo-swallow commit as it would have then been a forward-reference to the situation only in place after the following initramfs-updating commit which is better to keep separate. This is now quite a reasonable compromise.